torch_xla/csrc/tensor_methods.cpp

```diff
@@ -107,6 +107,8 @@ struct MinMaxValues {
 MinMaxValues GetMinMaxValues(const XLATensor& tensor,
                              c10::optional<at::Scalar> min,
                              c10::optional<at::Scalar> max) {
+  XLA_CHECK(min || max)
```

Comment:
Is this regexpd from tests?

Reply:
Yup it is. https://github.com/pytorch/pytorch/blob/master/test/test_torch.py#L8259